frontend/e-buddy/src/components/admin-panel/menu.tsx

```
@@ -28,129 +21,79 @@ export function Menu({ isOpen }: MenuProps) {
-        </ul>
-      </nav>
-    </ScrollArea>
+              </li>
+            ))}
+
+            {/* Logout Button */}
+            <li className="mt-6">
+              <button
+                onClick={logout}
+                className="flex items-center w-full px-3 py-2 hover:bg-accent2-light hover:text-white transition-colors duration-200"
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)